src/mlpack/methods/ann/augmented/tasks/copy_impl.hpp

```diff
@@ -69,7 +69,7 @@ inline void CopyTask::Generate(arma::field<arma::mat>& input,
     arma::colvec vecInput = arma::randi<arma::colvec>(
       size, arma::distr_param(0, 1));
     arma::colvec vecLabel = arma::conv_to<arma::colvec>::from(
-      arma::repmat(vecInput, nRepeats, 1));
+      repmat(vecInput, nRepeats, 1));
```

Comment:
```suggestion
        repmat(vecInput, nRepeats, 1));
```
Looks like there was a style "error" here and two lines above, feel free to fix it. :+1: